src/credentials/awsCredentialsStatusBarItem.ts

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as nls from 'vscode-nls'
+const localize = nls.loadMessageBundle()
+
+import * as vscode from 'vscode'
+import { AwsContext, ContextChangeEventsArgs } from '../shared/awsContext'
+
+const STATUSBAR_PRIORITY = 100
```

Comment:
Is there a reason why we're explicitly setting the priority?

Reply:
Probably not at this point. I preserved that from existing code.